src/test/java/core/basesyntax/service/DataHandlerImplTest.java

```diff
@@ -0,0 +1,59 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.impl.DataHandlerImpl;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class DataHandlerImplTest {
+    private static DataHandler dataHandler;
+    private static List<String> listForActual;
+    private static List<String> emptyList;
+    private static List<FruitTransaction> listForExpected;
+
+    @Before
+    public void setUp() {
+        dataHandler = new DataHandlerImpl();
+    }
+
+    @BeforeClass
+    public static void beforeClass() {
+        listForActual = new ArrayList<>();
+        listForExpected = new ArrayList<>();
+    }
+
+    @Test
+    public void dataHandler_validInput_isOk() {
```

Comment:
The method name is handleData
```suggestion
    public void handleData_validInput_isOk() {
```